internal/bindata/prepSetupStateroot.sh

```diff
@@ -42,7 +42,10 @@ function log_it {
 
 function build_kargs {
     jq -r '.spec.kernelArguments[]' "${img_mnt}/mco-currentconfig.json" \
-        | xargs --no-run-if-empty -I% echo -n "--karg-append % "
+        | while IFS= read -r line; do
```

Comment:
shouldnt it move to golang?

Reply:
yes, this is a temporary fix to keep it working with 4.14 until we move to golang